stable/node-problem-detector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: node-problem-detector
-version: "1.4.1"
+version: "1.42"
```

Comment:
Seems this should be `1.4.2` instead of `1.42`.

Reply:
Ah yes, typo in my rebase. Thanks.